sdk/textanalytics/Azure.AI.TextAnalytics/ToStringGuidence.md

```diff
@@ -0,0 +1,30 @@
+# ToString
+
+## Considerations
+- Most useful / main pain points
+- Following guidance from BCL team and .NET team members
+- Applied to TA and FR
+- No added logic into ToString. Plain and simple.
+
+## General Guidance
+- Can the type be parsed? -> Add ToString
```

Comment:
To add:
if it changes the behavior of the type. for example if the type authentication changes the exception or some functionalities, then the property should be added